setup.py

```diff
@@ -6,6 +6,19 @@
     r"""__version__ = ["']+([0-9\.]*)["']+""",
     open('openaerostruct/__init__.py').read(),
 )[0]
+
+optional_dependencies = {
+    'docs': ['openmdao[docs]==3.9.2'],
+    'test': ['testflo>=1.3.6']
+}
+
+# Add an optional dependency that concatenates all others

```

Comment:
Is this mentioned/used anywhere? Might be good to update docs or the GitHub Actions job to use this if suitable. If not, then don't worry about it, I just don't like adding code that's not actually used.

Reply:
We use `pip install .[all]` in Github Action. Also I'll add a note to docs or README

Reply:
Cool sounds good.